lib/config.js

```diff
@@ -3,13 +3,13 @@ module.exports = {
   headers: {
     'Content-Type': 'application/json',
     'User-Agent':
-      'Smsglobal-Node-SDK/1.0  (node ' +
+      'SMSGlobal-SDK/v2 Version/1.0.0 Node/' +
       process.version +
-      ') platform (' +
-      process.arch +
-      '-' +
+      ' (' +
       process.platform +
-      '; OpenSSL ' +
+      ' ' +
+      process.arch +
+      '; OpenSSL/' +
       process.versions.openssl +
       ')',
```

Comment:
Use a template string here. It will improve the readability. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals